### DISCUSSION TITLE
Questions about Premium Membership

### DISCUSSION THREAD

**Discussion by ByQwank** [KevinBytesTheDust/CVAmp#317] [Q&A]
Considering to purchase the Premium membership for the advaned beta features.
Would the Low CPU usage be able to handle 4-8 accounts with about 10 viewers?
How low CPU usage is it really?
I have a i7 8700k, 16gb of ram so I think im beyond sufficient.

**Comment by KevinBytesTheDust** [comment]
The low CPU option is really experimental, and I have not reconfirmed its effects for a long time. 
But with your CPU, you should easily reach your desired numbers with both the free and supporter version — your choice.

**Comment by Darkestangel69** [comment]
> Considering to purchase the Premium membership for the advaned beta features. Would the Low CPU usage be able to handle 4-8 accounts with about 10 viewers? How low CPU usage is it really? I have a i7 8700k, 16gb of ram so I think im beyond sufficient.

How you get account tokens? Making 100 Emails by hand takes forever DM me plz
